README.md

```diff
@@ -1 +1,237 @@
+free to use any package manager (either npm, yarn, pnpm...). Let's start by
+adding the SDK to your project:
+
+~~~bash
+# with yarn
+yarn add vocdoni-sdk
+# with npm
+npm i vocdoni-sdk
+# with pnpm
+pnpm add vocdoni-sdk
+~~~
+
+In case you want to create processes or vote to them, you'll need a signer to
+sign transactions. You should be able to use any kind of [ethers] signer.
+
+For this readme examples, the signer bootstraping will be ignored and you'll
+just see a `signer` constant.
+
+## SDK Usage
+
+The entry point is the SDK constructor, it will instantiate a new client
+connected to the specified API endpoint:
+
+~~~ts
+//                                   api endpoint URL                 signer
+const client = new VocdoniSDKClient('https://api-dev.vocdoni.net/v2', signer)
+~~~
+
+### Registering account
+
+Before creating any new processes, you should register your account against
+our blockchain (vochain):
+
+~~~ts
+(async () => {
+  const info = await client.createAccount()
+  console.log(info) // will show account information
+})();
+~~~
+
+The `createAccount` method will try to fetch an existing account first and, if
+it does not exist, it will register it against the blockchain.
+
+You can safely use `createAccount` to fetch any account information, but you can
+also decide to just fetch it, without falling back to an account registration:
+
+~~~ts
+(async () => {
+  const info = await client.fetchAccountInfo()
+  console.log(info) // shows info (only if account exists, otherwise throws error)
+})();
+~~~
+
+### Vocdoni tokens; faucet & balance
+
+Accounts require Vocdoni tokens in order to be able to register against our
+blokchain. The process above will automatically fetch some tokens from a faucet
+**under development**. For production environments, you should contact us (Vocdoni)
+for a byte64 string faucet, and specify it when creating your account:
+
+~~~ts
+(async () => {
+  const info = await client.createAccount({
+    faucetPayload: "<b64string>"
+  })
+})();
+~~~
+
+Accounts also require Vocdoni tokens in order to create new processes.
+
+You can check the balance thanks to the previous methods (`createAccount` and/or
+`fetchAccountInfo`) and, under development, you can request new tokens thanks
+to the included faucet:
+
+~~~ts
+(async () => {
+  const info = await client.createAccount()
+  if (info.balance === 0) {
+    await client.collectFaucetTokens()
+  }
+})();
+~~~
+
+> Note: the `collectFaucetTokens` method only works under development environment
+> and for accounts with not enough tokens to create new processes.
+
+### Creating a voting process
+
+After you successfully registered your account against Vocdoni's blockchain, you
+can start creating processes.
+
+A process will require a census of people who will vote it. Let's start creating
+it:
+
+~~~ts
+const census = new PlainCensus()
+census.add(address)
+// "address" is a valid ethereum-alike address
```

Comment:
Maybe it's worth mentioning that the census can also be created by using compressed public keys

```ts
census.add(computePublicKey(Wallet.createRandom().publicKey, true));
```

Reply:
Thanks, I'll add it too.